quickstart/deploy-mayastor.md

```diff
@@ -6,317 +6,99 @@
 Before deploying and using Mayastor please consult the [Known Issues](https://mayastor.gitbook.io/introduction/quickstart/known-issues) section of this guide.
 {% endhint %}
 
-In this Quickstart guide we demonstrate deploying Mayastor by using the Kubernetes manifest files provided within the project's repositories ([control plane components](https://github.com/openebs/mayastor-control-plane), [data plane components](https://github.com/openebs/mayastor)). These repositories are configured for the GitFlow release model, wherein the master branch contains official releases. By extension, the head of the master branch represents the latest official release.  Previous releases are identifiable by their annotated git tags.
+The steps and commands which follow are intended only for use with, and have only been tested against, the _latest release for version 2.x_. Earlier releases or development versions may require a modified or different installation process.
```

Comment:
```suggestion
The steps and commands which follow are intended only for use in conjunction with Mayastor version(s) 2.0.x.
```
We should aim to keep this branch (version/2.0) up to date and relevant to all patch versions of 2.0.  For 2.1 and beyond, there will be a new version of the documentation (e.g. version/2.1)

Reply:
Does make sense. updated it.

Reply:
Does make sense. updated it.